fract4d/c/model/STFractWorker.cpp

```diff
@@ -637,6 +633,33 @@ int STFractWorker::diff_colors(rgba_t a, rgba_t b)
     return dr * dr + dg * dg + db * db + da * da;
 }
 
+inline void STFractWorker::check_guess(int x, int y, rgba_t pixel)
+{
+    const calc_options &options = m_context->get_options();
+    const dvec4 pos = m_context->get_geometry().vec_for_point_2d(x, y);
+    rgba_t tpixel;
+    int titer;
+    float tindex;
+    fate_t tfate;
+    m_pf.calc(
+        pos.n,
+        options.maxiter,
+        periodGuess(),
+        options.period_tolerance,
+        options.warp_param,
+        x, y, 0,
+        &tpixel, &titer, &tindex, &tfate);
+    if (tpixel == pixel)
+    {
+        m_stats.s[PIXELS_SKIPPED_RIGHT]++;
```

Comment:
Prefer `++m_stats.s[PIXELS_SKIPPED_RIGHT];` - compiler will have less work to do and debug will run faster.